app/_includes/plugins/ai-proxy/templating.md

```diff
@@ -4,16 +4,39 @@
 {% assign options = include.params.options %}
 {% assign options_slug = options | slugify | prepend: 'schema--' %}
 
-The plugin enables you to substitute values in the [`{{ model_name }}`](./reference/#{{ model_name_slug }}) and any parameter under [`{{ options }}`](./reference/#{{ options_slug }})
-with specific placeholders, similar to those in the [Request Transformer Advanced](/plugins/request-transformer-advanced/)
-templating system.
+The plugin allows you to substitute values in the [`{{ model_name }}`](./reference/#{{ model_name_slug }}) and any parameter under [`{{ options }}`](./reference/#{{ options_slug }})
+with specific placeholders, similar to those in the [Request Transformer Advanced](/plugins/request-transformer-advanced/) plugin.
 
 The following templated parameters are available:
 
-* `$(headers.header_name)`: 
-* `$(uri_captures.path_parameter_name)`
-* `$(query_params.query_parameter_name)`
+* `$(headers.header_name)`: the value of a specific request header.
+* `$(uri_captures.path_parameter_name)`: the value of a captured URI path parameter.
+* `$(query_params.query_parameter_name)`: the value of a query string parameter.
 
-This can be used to OpenAI-compatible SDK with this plugin in multiple ways, depending on the required use case.
+You can combine these parameters with an OpenAI-compatible SDK in multiple ways using the AI Proxy Advanced plugin, depending on your specific use case:
```

Comment:
Is this technically possible with the AI Proxy plugin as well? Or do we want to direct people to AI Proxy Advanced either way?